### YamlMime:UniversalReference
api_name: []
items:
- children:
  - azure.mgmt.resourcegraph.models.facet_error_py3.FacetError
  fullName: azure.mgmt.resourcegraph.models.facet_error_py3
  langs:
  - python
  module: azure.mgmt.resourcegraph.models.facet_error_py3
  name: facet_error_py3
  type: module
  uid: azure.mgmt.resourcegraph.models.facet_error_py3
references:
- fullName: azure.mgmt.resourcegraph.models.facet_error_py3.FacetError
  isExternal: false
  name: FacetError
  parent: azure.mgmt.resourcegraph.models.facet_error_py3
  uid: azure.mgmt.resourcegraph.models.facet_error_py3.FacetError
